<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Converter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>yt.png</normaloff>yt.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Mp4</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Youtube Video Converter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Video Url : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1080p HD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>720p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>480p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>360p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>240p</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>144p</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert and Download</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>60 FPS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30 FPS</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Download location</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>296</x>
        <y>250</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mp3</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Youtube Audio Converter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Video Url : </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert and Download</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>190</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Download location</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>296</x>
        <y>250</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="action_4">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mp4</string>
   </property>
  </action>
  <action name="actionMp3">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mp3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
